src/player/player_update.cpp

```diff
@@ -4590,3 +4639,42 @@ void UpdatePlayer()
 
     NPCQueues::PlayerTemp.clear();
 }
+
+void UpdatePlayerPhysics() //Basically playerPhysicsPatch.lua by Emral, now on TheXTech!
+{
+    if(g_compatibility.easier_player_physics)
+    {
+        for(int tmpNumPlayers = numPlayers, A = 1; A <= tmpNumPlayers; A++)
+        {
+            Player[A].lastXSpeed = Player[A].lastXSpeed or 0;
+            if(!Player[A].Slide)
+            {
+                if((!PlayerGroundTouching(A) && Player[A].Duck))
+                {
+                    int mod = 1;
+                    if(!PlayerGroundTouching(A))
+                        mod = 2;
+                    if(Player[A].Controls.Right)
+                    {
+                        if(Player[A].Location.SpeedX < 0)
+                            Player[A].Location.SpeedX = Player[A].Location.SpeedX + 0.08 * mod;
+                    }
+                    else if(Player[A].Controls.Left)
+                    {
+                        if(Player[A].Location.SpeedX > 0)
+                            Player[A].Location.SpeedX = Player[A].Location.SpeedX - 0.08 * mod;
+                    }
+                    else
+                        Player[A].Location.SpeedX = Player[A].Location.SpeedX * 0.98;
+                    int xSpeedDiff = Player[A].Location.SpeedX * Player[A].lastXSpeed;
```

Comment:
Needed a blank line between these lines. You should add blank lines to make the code visually look better. If you don't put blank lines where they are needed, code looks like a clutter of text. So, it's better to logically separate if blocks, and various logical parts of code.